CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ All notable changes to this project will be documented in this file.
 * ### ALL
     * #### Added
     * #### Changed
+        * Fix [#2069](https://github.com/ni/nimi-python/issues/2069)
```

Comment:
Honestly, I'm a bit torn about whether to mention this in the changelog. It's not a behavioral change in the API and I don't **think** it will affect the installation experience, but if someone is installing with setuptools, who knows?

Reply:
Let's keep.